python/mxnet/metric.py

```diff
@@ -20,15 +18,22 @@ def check_label_shapes(labels, preds, shape=0):
                          "predictions {}".format(label_shape, pred_shape))
 
 class EvalMetric(object):
-    """Base class of all evaluation metrics."""
+    """Base class of all evaluation metrics.
+
+    .. note::
+
+        This is a base class that providing common metric interfaces.
+        You should not use it directly. You can create new metric classes on top
```

Comment:
Suggest changing to:

One should not use this class directly, but instead create new metric classes that extend it.